brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3103: Fix: CTA additional button padding issue fix in frontend

### Description

[CTA] Additional Button CTA Spacing Responsive not working in frontend
https://brainstormforce.atlassian.net/browse/SPCTR-3103

### Screenshots

https://d.pr/v/SUWByx

### Types of changes





### How has this been tested?


- Add CTA Block.
- Add Additional Button in it.
- Now try to add padding of Additional Button spacing from the Styling panel for all devices and check in frontend.

### Checklist:
